classes/Kohana/Model/Purchase.php

```diff
@@ -106,9 +106,9 @@ public function find_brand_purchase(Model_Brand $brand)
 	 */
 	public function add_item($brand, Model_Purchase_Item $new_item)
 	{
-		$brand_purchase = $this->find_or_build_brand_purchase($brand);
-		$brand_purchase->add_or_update_item($new_item);
-		$new_item->brand_purchase = $brand_purchase;
+		$this
+			->find_or_build_brand_purchase($brand)
```

Comment:
@phgeorgiev I have reverted the changes. Would you make an issue to fix indentation?